CONTRIBUTING.md

```diff
@@ -5,19 +5,64 @@ The Python special interest group (SIG) meets regularly. See the OpenTelemetry
 information on this and other language SIGs.
 
 See the [public meeting notes](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)
-for a summary description of past meetings. To request edit access join the
-meeting or get in touch on Gitter.
+for a summary description of past meetings. To request edit access, join the
+meeting or get in touch on [Gitter](https://gitter.im/open-telemetry/opentelemetry-python).
 
-## Running Tests
+## Pull Request
 
-Execute tests as well as the linting process with tox:
+### How to send PR
 
-    pip install --user tox
-    tox  # execute in the root of the repository
+* Everyone can send PR.
+* You need to fork the project in GitHub.
 
-## Testing
+Clone the upstream repo:
 
-The standard Python unittest module is used to author unit tests.
+```sh
+$ git clone https://https://github.com/open-telemetry/opentelemetry-python.git
+```
+
+Add your fork as an origin:
+
+```sh
+$ git remote add fork https://github.com/YOUR_GITHUB_USERNAME/opentelemetry-python.git
+```
+
+Run tests:
+
+```sh
+# make sure you have all supported versions of Python installed
+$ pip install tox  # only first time.
+$ tox  # execute in the root of the repository
+```
+
+Checkout a new branch, make modifications and push the branch to your fork:
+
+```sh
+$ git checkout -b feature
+# edit files
+$ git commit
+$ git push fork feature
+```
+
+Open a pull request against the main opentelemetry-python repo.
+
+### How to Receive Comments
+
+* If the PR is not ready for review, please put "[WIP]" in the title, or tag it
+  as "work-in-progress", or mark it as "draft".
+* Make sure CLA is signed and CI is clear.
+
+### How to Get PR Merged
+
+A PR is considered as **ready for merge** when:
+* It got two approvals from Collaborators/Maintainers (at different companies).
+* Major feedbacks should be resolved.
+* The PR should stay at least one working day before getting merged, this
+  gives people reasonable time to review.
+* Trivial change (typo, cosmetic, doc, etc.) doesn't have to wait for one day.
+* Urgent fix can take exception as long as it has been actively communicated.
+
+Any Collaborator/Maintainer can merge the PR once it is **ready for merge**.
```

Comment:
I need help on reviewing my English wording/phrasing.
e.g. should this be **ready to merge** or **ready for merge**?

Reply:
I'd say "ready to be merged".

Reply:
Decided to follow the spec repo tag:

![image](https://user-images.githubusercontent.com/17327289/63184205-ac55fb00-c00b-11e9-9ba4-e793f6d01548.png)